version_control/katalyst_by_codurance/templates/layouts/base.html

```diff
@@ -11,6 +11,9 @@
     {{ require_css(get_asset_url("../../css/main.css")) }}
     {{ require_css(get_asset_url("../../css/theme-overrides.css")) }}
     {{ standard_header_includes }}
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700;800&display=swap" rel="stylesheet">
```

Comment:
I think I've mentioned this to you, but loading fonts this way it will track users information from our website from google. I might have a look and see what's the best way to load variable fonts